_data/rules.yml

```diff
@@ -1300,6 +1300,74 @@ groups:
+                severity: critical
+              - name: ClickHouse Disk Space Low on Backups
+                description: Disk space on backups is below 20%.
+                query: "ClickHouseAsyncMetrics_DiskAvailable_backups / (ClickHouseAsyncMetrics_DiskAvailable_backups + ClickHouseAsyncMetrics_DiskUsed_backups) * 100 < 20"
+                severity: warning
+              - name: ClickHouse Replica Errors
+                description: Critical replica errors detected, either all replicas are stale or lost.
+                query: "ClickHouseErrorMetric_ALL_REPLICAS_ARE_STALE == 1 or ClickHouseErrorMetric_ALL_REPLICAS_LOST == 1"
+                severity: critical
+              - name: ClickHouse No Available Replicas
+                description: No available replicas in ClickHouse.
+                query: "ClickHouseErrorMetric_NO_AVAILABLE_REPLICA == 1"
+                severity: critical
+              - name: ClickHouse No Live Replicas
+                description: There are too few live replicas available, risking data loss and service disruption.
+                query: "ClickHouseErrorMetric_TOO_FEW_LIVE_REPLICAS == 1"
+                severity: critical
+              - name: ClickHouse High Network Traffic
+                description: Network traffic is unusually high, may affect cluster performance.
+                query: "ClickHouseMetrics_NetworkSend > 1000 or ClickHouseMetrics_NetworkReceive > 1000"
+                severity: warning
+                comments: |
+                  Please replace the threshold with an appropriate value
+              - name: ClickHouse High TCP Connections
+                description: High number of TCP connections, indicating heavy client or inter-cluster communication.
+                query: "ClickHouseMetrics_TCPConnection > 1500"
+                severity: warning
+                comments: |
+                  Please replace the threshold with an appropriate value
+              - name: ClickHouse Interserver Connection Issues
+                description: An increase in interserver connections may indicate replication or distributed query handling issues.
+                query: "increase(ClickHouseMetrics_InterserverConnection[5m]) > 0"
+                severity: warning
+              - name: ClickHouse ZooKeeper Connection Issues
+                description: ClickHouse is experiencing issues with ZooKeeper connections, which may affect cluster state and coordination.
+                query: "avg(ClickHouseMetrics_ZooKeeperSession) != 1"
+                severity: warning
+              - name: ClickHouse Authentication Failures
+                description: Authentication failures detected, indicating potential security issues or misconfiguration.
+                query: "increase(ClickHouseErrorMetric_AUTHENTICATION_FAILED[5m]) > 0"
+                severity: critical
```

Comment:
i would use "warning" or "info" instead

A critical alert must be checked early, this rule is "just" a security notification.

WDYT ?